### DISCUSSION TITLE
Question about Axis C1410

### DISCUSSION THREAD

**Discussion by Ido220694** [AxisCommunications/discussions#249] [General]
I'm trying to play audio file through G-streamer from remote computer.
I have a valid ip to axis c14010 and have communication with the device.  

I tried to run from the terminal:
gst-launch-1.0 -v filesrc location="path-to-audio-file" ! audioconvert ! audioresample ! opusenc ! rtpopuspay ! udpsink host="ip-to-axis-c1410" port=5004

but it didn't seems to work. How can I play audio-files from remote? 
Also I want to communicate with the device from remote, for 2 way communication. How can I do it from remote?

Thamks

**Comment by vivekatoffice** [comment]
Related link:
[308](https://github.com/orgs/AxisCommunications/discussions/308)
[34](https://github.com/orgs/AxisCommunications/discussions/34)
